backend/de.metas.util.web/src/main/java/de/metas/util/web/audit/ApiAuditHouseKeepingTask.java

```diff
@@ -46,7 +48,13 @@ public ApiAuditHouseKeepingTask(
 	@Override
 	public void executeTask()
 	{
-		final ImmutableList<ApiRequestAudit> timeWiseSortedApiRequests = apiRequestAuditRepository.getAllFailingTimeWiseSorted();
+		final ApiRequestQuery apiRequestQuery = ApiRequestQuery.builder()
+				.apiRequestStatusSet(ImmutableSet.of(Status.ERROR))
```

Comment:
please add status= "received here as well" (=> the other issue i just added is probably done then)